### DISCUSSION TITLE
Access to GPT-4

### DISCUSSION THREAD

**Discussion by HavenDV** [Azure-Samples/azure-ai#3] [Q&A] [answered]
Hello. I represent a small startup from Dubai that is building a bot based on GPT-4. I would like to know if there is a way to use Azure OpenAI Service and GPT-4 if we still don't have an open official company (it's overhead for a startup at our stage)? We are also interested in the possibility of obtaining a grant for a startup with the same issue.

**Comment by achandmsft** [comment, marked as answer]
Hi @HavenDV, thanks for your question. Could you please shoot me an email (achand at microsoft dot com)? I will loop you in with the correct folk who can help you.
